assets/css/abstracts/_mixins.scss

```diff
@@ -75,3 +75,18 @@
 	margin: unset;
 	overflow: hidden;
 }
+
+// Reset <button> style so we can use link style for action buttons.
+@mixin link-button() {
+	margin: 0;
+	padding: 0;
+	border: 0;
+
+	font-size: inherit;
+	font-weight: normal;
+
+	background: transparent;
```

Comment:
This makes me think I would love to have a lint rule or a tool like CSS comb so we always order CSS properties in the same way. Is that opinion shared by other members of the team? Should we open an issue?

Reply:
I like this, I'm currently defaulting to "alphabetic unless I change my mind" :)